src/main/java/eli/command/ListCommand.java

```diff
@@ -0,0 +1,19 @@
+package eli.command;
+
+import eli.storage.Storage;
+import eli.task.TaskList;
+import eli.ui.Ui;
+
+/**
+ * Command to list the application.
+ */
+public class ListCommand extends Command {
+
+  /**
+   * Constructor for ListCommand.
+   */
+  @Override
+  public void execute(TaskList tasks, Ui ui, Storage storage) {
+    tasks.list();
+  }
```

Comment:
I noticed that other Commands are calling the ui to print out the responses. Would it be better if this was the same as well?